packages/node_modules/@ciscospark/react-redux-spark/src/spark.js

```diff
@@ -24,71 +90,32 @@ export default function createSpark(accessToken, options = {}) {
           access_token: accessToken || credentials.access_token
         }
       },
-      config: {
-        appName: 'spark-widgets',
-        appVersion: process.env.REACT_CISCOSPARK_VERSION,
-        device: {
-          ephemeral: true
-        },
-        logger: {
-          level: options.logLevel || process.env.NODE_ENV === 'production' ? 'silent' : 'trace'
-        },
-        conversation: {
-          allowedInboundTags: {
-            'spark-mention': ['data-object-type', 'data-object-id', 'data-object-url'],
-            a: ['href'],
-            b: [],
-            blockquote: ['class'],
-            strong: [],
-            i: [],
-            em: [],
-            pre: [],
-            code: [],
-            br: [],
-            hr: [],
-            p: [],
-            ul: [],
-            ol: [],
-            li: [],
-            h1: [],
-            h2: [],
-            h3: []
-          },
-          allowedOutboundTags: {
-            'spark-mention': ['data-object-type', 'data-object-id', 'data-object-url'],
-            a: ['href'],
-            b: [],
-            blockquote: ['class'],
-            strong: [],
-            i: [],
-            em: [],
-            pre: [],
-            code: [],
-            br: [],
-            hr: [],
-            p: [],
-            ul: [],
-            ol: [],
-            li: [],
-            h1: [],
-            h2: [],
-            h3: []
-          }
-        },
-        // Added to help load blocking during decryption
-        encryption: {
-          kmsInitialTimeout: 10000
-        },
-        phone: {
-          enableExperimentalGroupCallingSupport: true
-        },
-        storage: {
-          unboundedAdapter: new LocalForageStoreAdapter('ciscospark-widgets')
-        }
-      }
+      config: defaultConfig(options)
     });
     window.ciscosparkInstance = ciscosparkInstance;
   }
 
-  return ciscosparkInstance;
+  return Promise.resolve(ciscosparkInstance);
+}
+
+
+export function createSparkJwt(jwt, options = {}) {
+  // Check if we already have a spark instance in memory
+  let {ciscosparkInstance} = window;
+
+  if (typeof ciscosparkInstance === 'object') {
+    return Promise.resolve(ciscosparkInstance);
+  }
+
+  const credentials = JSON.parse(localStorage.getItem('credentials') || false);
+  if (credentials.access_token) {
+    return Promise.resolve(createSpark(credentials.access_token, options));
+  }
```

Comment:
Are we sure we want to use a stored accessToken to authenticate even when someone passes a JWT? Is it possible for another user's credentials to get cached and when a JWT is passed, the old/different user credentials are used?

Reply:
Ah yea, good catch

Reply:
Should we remove that feature from the regular access token login method too? Since we're using `LocalForageStoreAdapter` we probably shouldn't manually inspect localstorage at all for spark credentials